docs/day-two/quality-of-life-tweaks.md

```diff
@@ -13,6 +13,13 @@ alias dcp='docker-compose -f ~/docker-compose.yml '
 !!! info
     If you have a useful alias you think would benefit PMS users please submit a PR via GitHub for inclusion in this site following the format below. One alias or a group of related aliases showing example output.
 
+### Aliases in fish shell
+If fish shell is being used, aliases created using the `alias` command will not persist from one session to the next. In order to save an alias the `-s` or `--save` option must be used, refer to fish shell documentation: [alias - create a function](https://fishshell.com/docs/current/cmds/alias.html). Using the same example as above creating a persistent an alias in Fish looks like this:
```

Comment:
Aliases in bash nor zsh persist between sessions. The way to handle this is to put in `.bashrc` / `.zshrc` / fish equivalent. 

Might be better to just note that aliases don't persist, and that to make them persist they should go in your shells' `.rc` file?

Reply:
I didn't know that, makes sense. I am happy to rewrite something to this effect.
What is the best practise here? Do I just close this pull request and submit a new one or edit this one?

Reply:
Whatever you feel most comfy with. However, I'd say we keep this one open as we'll be able to see the context more easily. Or, you could open a new one and reference it with #16 (pound number) and GitHub will auto link the two.

Reply:
Ok! I think i figured it out changes made